plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -51,6 +51,16 @@ public static function load() {
 		do_action( 'woocommerce_helper_loaded' );
```

Comment:
All of the changes in this file is to support different redirect_uri - either from wc-addons (Extensions) or wc-admin (Home)